trio/_core/_io_kqueue.py

```diff
@@ -149,7 +148,7 @@ def abort(raise_cancel: RaiseCancelT) -> Abort:
         # wait_task_rescheduled does not have its return type typed
         return await _core.wait_task_rescheduled(abort)  # type: ignore[no-any-return]
 
-    async def _wait_common(self, fd: int | socket, filter: int) -> None:
+    async def _wait_common(self, fd: int | _HasFileNo, filter: int) -> None:
```

Comment:
I'm a bit weirded out by codecov because this *should* be covered.

Reply:
on a rerun there were no coverage warning, so seems like a temporary glitch.